truffle/src/main/java/org/jruby/truffle/runtime/core/CoreLibrary.java

```diff
@@ -128,6 +128,7 @@
     private final RubyModule rubiniusFFIModule;
     private final RubyModule signalModule;
     private final RubyModule truffleModule;
+    private final RubyClass bigDecimalClass; // TODO (pitr 15-May-2015): it's not part of core but stdlib
```

Comment:
It's fine that this lives in `CoreLibrary`. We have all sorts of stuff in here - it's really any Ruby classes provided by this implementation.